Duo_S/NuttX/README.md

```diff
@@ -22,13 +22,14 @@ last_update: 2024-06-21
 ### Hardware Information
 
 - Milk-V Duo S (512M, SG2000)
-- A USB Power Adapter
-- A USB-A to C or USB C to C cable for powering the development board
-- A microSD card
-- A USB card reader
-- A USB to UART Debugger (e.g., CP2102, FT2232, etc. Be aware that WCH CH340/341 series will cause garbled text output, DO NOT USE)
-- Three DuPont wires
-- Ethernet access for TFTP Boot
+- A USB-A to C or USB C to C cable to power the development board
+- A microSD card for booting
+- A USB card reader for microSD burning
+- A USB to UART debugger (such as: CP2102, FT2232, etc., garbled characters will appear when using the CH340/341 series, which is an expected phenomenon)
+- Three Dupont lines for uart interface
+- Network cabling for optical fiber access (TFTP Boot cannot use USB network)
```

Comment:
What do you mean optical fiber?
There's no such port on the Duo S. Nor needed by NuttX.

Reply:
Translation error.I will be caerful with translation software.

Reply:
Translation error
I will be careful with translation software